build/index.js

```diff
@@ -35,124 +42,109 @@ const bootstrap = () => {
     if (!fs.existsSync(elmJsonFile)) {
       generateElmJson();
     }
-    return Promise.resolve();
   } catch (e) {
-    return Promise.reject(`Could not complete bootstrap: ${e}`);
+    return new Error(`Could not complete bootstrap: ${e}`);
```

Comment:
returning the error won't make the promise reject will it? you need to throw or return a rejected promise, otherwise you end up with a Promise that fulfills with an error. :-)

Reply:
Should have been a throw. Will fix that

Reply:
Reverted back to returning a Promise.